### PR TITLE
Subquery operators

### PR DESCRIPTION
**Pull request by zdikov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add support for INTERSECT, EXCEPT, ANY, ALL operators.

### CODE REVIEW

**Comment by CurtizJ** [comment]
Please, don't forget about adding tests.

File: src/Parsers/ExpressionListParsers.cpp [hunk 1]
```diff
@@ -168,6 +171,172 @@ static bool parseOperator(IParser::Pos & pos, const char * op, Expected & expect
...
+
+    auto select_with_union_query = std::make_shared<ASTSelectWithUnionQuery>();
+    auto list_of_selects = std::make_shared<ASTExpressionList>();
+    list_of_selects->children.push_back(select_query);
+    select_with_union_query->list_of_selects = list_of_selects;
+    select_with_union_query->children.push_back(select_with_union_query->list_of_selects);
+
+    auto new_subquery = std::make_shared<ASTSubquery>();
+    new_subquery->children.push_back(select_with_union_query);
+    function->children[0]->children.pop_back();
```
Comment (danlark1) [inline src/Parsers/ExpressionListParsers.cpp:233]: Why not just std::move to the last?

**Review by kssenii — APPROVED** [review]

**Comment by sevirov** [comment]
Internal documentation ticket: DOCSUP-13264

File: src/AggregateFunctions/AggregateFunctionMinMaxAny.h [hunk 2]
```diff
@@ -975,6 +982,68 @@ struct AggregateFunctionAnyLastData : Data
...
+        {
+            first_value = false;
+            this->change(column, row_num, arena);
+            return true;
+        }
+        else if (!this->isEqualTo(column, row_num))
+        {
+            is_null = true;
+        }
+        return false;
```
Comment (tavplubix) [inline src/AggregateFunctions/AggregateFunctionMinMaxAny.h:1007]: So it returns false when the value was actually changed to NULL, why? @kssenii, do you remember?

**Pull request merged**